docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -58,34 +61,57 @@ Maintainers of a translation are expected to review translations for quality, fr
 
-### Maintainer responsibilities
+## Maintainer responsibilities
 
 As repo maintainers and members of the Gatsby community, your responsibilities are as follows:
 
-- Keep issues up to date as people volunteer to translate pages.
+- Keep issues up-to-date as people volunteer to translate pages.
 - Review pull requests made by contributors promptly.
-- Review pull requests generated by gatsbybot in order to make sure translations remain up to date with the source repo. (details to come)
+- Review pull requests generated by gatsbybot in order to make sure translations remain up-to-date with the source repo. (details to come)
 - Act as point of contact for your language and answer questions from both contributors to your language and the core Gatsby team.
 - Set up a process in order to get your translation published. (details to come)
 
 As a maintainer, you are welcome to add a contributing doc written in your language to assist with the process. You can find an example in the [gatsby-es repo](https://github.com/gatsbyjs/gatsby-es/blob/master/CONTRIBUTING.MD). Translating [this page](https://github.com/gatsbyjs/gatsby/blob/master/docs/contributing/gatsby-docs-translation-guide.md) and copying it into a `contributing.md` file would be an option as well.
 
-Additional tips:
+### Additional tips
+
+#### Set up a style guide and glossary
+
+Your language repo comes with a template [style guide](https://github.com/gatsbyjs/gatsby-i18n-source/blob/master/style-guide.md) that you can use to put in style rules specific to your language. Refer to the [translation style guide](#translation-style-guide) section for more information.
+
+#### Collaborate on difficult pages
+
+Some pages are very long and difficult to translate for a single contributor. Feel free to split these pages up into sections and work on them together!
+
+In addition, sometimes a contributor cannot continue a PR for whatever reason. It may help to ask them if they need assistance and to bring someone else on to complete the work in a timely fashion.
+
+#### Set up a review process
+
+As codeowners, you have the freedom and responsibility to decide what your review process will be like. You can decide how many reviewers you'd like. If your team is small, one reviewer may be enough. But if you have lots of contributors and enough codeowners, you may want to require two reviewers for additional quality.
+
+You have the ability to install any plugin or automation tool that will make your life easier as codeowners. Is there a text linter that works well in your language? Is there any automation that you can add? If you feel that these improvements would be beneficial to other languages as well, create an issue or PR for it in the main Gatsby repository.
```

Comment:
One question: are there any limits on what they can install?

Reply:
Does gatsby have any restrictions on what apps can be installed or what packages can be added as dependencies? Also, any new github plugins have to go through the gatsby admins before they get installed.

I guess I should change this to:

> You are empowered to install plugins and automation tools that will make your life easier as codeowners. [...] Create an issue/PR and tag @gatsby-core so we can review the use case. If you feel that these improvements would be beneficial...

Reply:
Those are mostly questions for @gatsbyjs/core in terms of package/app restrictions, but that seems like a good idea to have contributors open an issue and tag @gatsby-core to vet stuff.

Reply:
We can merge this in as-is and make changes to it later if there ends up being things requested that we don't want to approve. Thanks for adding this in there!